tools/mason/MasonTest.chpl

```diff
@@ -581,12 +578,12 @@ proc addTestResult(ref result, ref dictDomain, ref dict, ref testNames,
     when "IncorrectNumLocales" {
       if comm != "none" {
         var strSplit = errMsg.split("=");
-        var reqLocalesStr = strSplit[2].strip().split(" ");
+        var reqLocalesStr = strSplit[2].strip().split(",");
         for a in reqLocalesStr do
-          if dictDomain.contains(a: int) then
-            dict[a: int] += 1;
+          if localesCountMap.contains(a: int) then
```

Comment:
This would be cleaner to do the cast once:

```
const locIdx = a:int;
  if localesCountMap.contains(locIdx) then
    localesCountMap[locIdx] += 1;
  else
    ...

```